docs/install.md

```diff
@@ -165,5 +165,5 @@ all the nodes running `dacd`.
 On the compute nodes, it seems possible to restrict sudo access to the
 following:
 ```
-dacd ALL=(ALL) NOPASSWD: /usr/bin/mkdir -p /dac/*, /usr/bin/chmod 770 /dac/*, /usr/bin/chmod 0600 /dac/*, /usr/bin/chown * /dac/*, /usr/bin/mount -t lustre * /dac/*, /usr/bin/umount -l /dac/*, /usr/bin/umount /dac/*, /usr/sbin/losetup /dev/loop* /dac/*, /usr/sbin/losetup -d /dev/loop*, /usr/sbin/mkswap /dev/loop*, /usr/sbin/swapon /dev/loop*, /usr/sbin/swapoff /dev/loop*, /usr/bin/ln -s /dac/* /dac/*, /usr/bin/dd if=/dev/zero of=/dac/*, /usr/bin/rm -df /dac/*, /bin/grep /dac/* /etc/mtab
```

Comment:
we don't currently use umount -l, after it hid problems with compute nodes unmounting cleanly.